src/state.rs

```diff
@@ -166,6 +200,11 @@ impl eframe::App for State {
         // Run self.update() on each manager
         let _ = &self.update_managers();
 
+        // TODO(orkaboy): Should probably not be here
+        if self.context.process.is_some() {
+            let _ = self.sequencer.run(&mut self.game_state);
```

Comment:
unless your sequencer does not relinquish control this is actually an ok place to put it until we build the game manager.